app/services/ValidatingPageBuilder.scala

```diff
@@ -63,7 +63,7 @@ class ValidatingPageBuilder @Inject() (pageBuilder: PageBuilder){
         duplicateUrlErrors(pages.reverse, Nil) ++
         checkDateInputErrorCallouts(pages, Nil) ++
         checkExclusiveSequenceTypeError(pages, Nil) ++
-        checkExclusiveSequencePages(pages, Nil) ++
+        //checkExclusiveSequencePages(pages, Nil) ++
```

Comment:
If this has been removed from original lines 152 -166 then I guess that is why this is commented out here - however it should probably go if it ceases to be available to call ?

Reply:
will remove

Reply:
Verified as removed